packages/amd-k8s-device-plugin/amd-k8s-device-plugin.spec

```
@@ -0,0 +1,75 @@
+%global goproject github.com/ROCm
+%global gorepo k8s-device-plugin
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.31.0.8
+%global rpmver %{gover}
+
+Name: %{_cross_os}amd-k8s-device-plugin
```

Let's keep this consistent with the upstream naming 
```suggestion
Name: %{_cross_os}rocm-k8s-device-plugin
```

---

Makes sense. Will update.